### DISCUSSION TITLE
Website for trips

### DISCUSSION THREAD

**Discussion by kyrreso** [Norsk-institutt-for-samkjoring/Standard#3] [Ideas]
This is interesting https://ride2go.com/?trip=255574.
We should also have a way to get webpages to display trips

**Comment by MartinH-open** [comment]
Yes, sure. Planned trips and their paths should be displayed at wish. 
Also if a trip is already underway the realtime postion (or a calculated one) of the car should be displayed on the map. Eventually display an image of the car or pictures of the passengers if they agreed.
Depending on the map size and the number of offers at this point in time the more or less rides should be displayed. For a certain user with some direction in mind only the related / matching ride offers should be displayed with some filtering function.
